.ipython/profile_default/startup/00-prelude.ipy

```diff
@@ -0,0 +1,30 @@
+%load_ext autoreload
+%autoreload 2
+from zonos.model import Zonos
+import torch
+from zonos.conditioning import make_cond_dict
+from zonos.model import ZonosDecodeOne
+
+model = Zonos.from_pretrained("Zyphra/Zonos-v0.1-transformer", device="cuda", backbone="torch")
+input_ids = torch.zeros((1, 20, 1), dtype=torch.int64, device="cuda")
+cond_dict = make_cond_dict(text="Hello, world! I am the one you need. Let me take you to a far off place.", language="en-us")
+prefix_conditioning = model.prepare_conditioning(cond_dict)
+batch_size = 1
+audio_prefix_codes = None  # No prefix for now
+prefix_audio_len = 0 if audio_prefix_codes is None else audio_prefix_codes.shape[2]
+max_new_tokens: int = 86 * 30
+audio_seq_len = prefix_audio_len + max_new_tokens
+seq_len = prefix_conditioning.shape[1] + audio_seq_len + 9
+device = 'cuda'
+with torch.device(device):
+    inference_params = model.setup_cache(batch_size=batch_size * 2, max_seqlen=seq_len)
```

Comment:
When I was debugging, I was repeatedly running generate() from an ipython session while editing model.py (e.g., toggling the `and False` I had in there.) This will setup some useful variables in your context for calling generate. You can use it with IPYTHONDIR=.ipython ipython